modules/juce_audio_devices/native/juce_android_Audio.cpp

```diff
@@ -100,8 +119,8 @@ class AndroidAudioIODevice  : public AudioIODevice,
         inputSampleRate = 44100;
         minBufferSizeIn = env->CallStaticIntMethod(AudioRecord, AudioRecord.getMinBufferSize, inputSampleRate,
             CHANNEL_IN_MONO, ENCODING_PCM_16BIT) / 2;
-        numDeviceInputChannelsAvailable = minBufferSizeIn > 0 ? 1 : 0;
-
+        numDeviceInputChannelsAvailable = minBufferSizeIn > 0 ? 1 : 0;

```

Comment:
This change seems to be a matter of inconsistent line endings in the original file.  I didn't know how to remove this change with the editors that I have on my machine.